src/components/Articles.jsx

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from "react";
+import { getArticles } from "../utils/api.js";
+import ArticleCard from "./ArticleCard";
+
+const Articles = () => {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    getArticles().then((data) => {
+      setArticles(data);
+    });
+  }, []);
+
+  return (
+    <main>
+      <ul>
+        {articles.map((article) => {
+          return (
+            <div className="article-card">
+              <a href={`/articles/${article.article_id}`}>
+                <li key={article.article_id}>
+                  <ArticleCard article={article} />
+                </li>
+              </a>
+            </div>
+          );
+        })}
+      </ul>
```

Comment:
For accessibility purposes, a `ul` element should only have `li` as its children, whereas here the children of your `ul` are `div`s. The `li` itself can contain whatever you want however, so feel free to put everything else inside it!

Additionally, whilst an `a` tag will do a successful job of routing the user to the right, it will cause the whole app to refresh which is needlessly slow. Make use of the `Link` component that we can import from react router!